src/main.py

```diff
@@ -262,10 +262,10 @@ def _build_local_images(target_version: Version, target_ecr_repo_list: list[str]
                 for t in image_tags_to_apply:
                     image.tag(target_ecr_repo, tag=t)
                     generated_image_versions.append({'repository': target_ecr_repo, 'tag': t})
-        else:
-            # Tag the image for testing
-            image.tag('localhost/sagemaker-distribution',
-                      config['image_tag_generator'].format(image_version=str(target_version)))
+
```

Comment:
Here changed it to tagging `localhost/sagemaker-distribution` every time, even when pushing to repo. This tag is required to run tests, so this allows us to build, push, and test in one step. Previously needed either --target-ecr-repo or --skip-tests.